workorder-job-repeater.lua

```diff
@@ -14,6 +14,23 @@ work orders). This script makes the dwarfes work harder.
 For best experience add following to your ``onMapLoad*.init``::
 
     workorder-job-repeater start
+    
+**Usage**:
+
+``[enable | disable] workorder-job-repeater``
+
+``workorder-job-repeater start [<frequency>] | stop``
+
+``workorder-job-repeater --frequency <ticks> | --verbose | --help``
+
+:enable, start <frequency>: starts job monitoring, running every ``<frequency>`` ticks (default: 10).
+:disable, stop:             stops the script.
+
+-f, --frequency <ticks>     sets job monitoring frequency to ``<ticks>``
+                            (only if it's lower than the previously set value,
```

Comment:
After DFHack/dfhack#1876 is merged, you will be able to adjust either up or down